util-images/watch-list/Makefile

```diff
@@ -0,0 +1,17 @@
+PROJECT = k8s-staging-perf-tests
+IMG = gcr.io/$(PROJECT)/watch-list
+TAG = v0.0.1
+
+all: push
+
+.PHONY: build
+build:
+	docker build --pull -t $(IMG):$(TAG) .
+	docker tag $(IMG):$(TAG) $(IMG):latest
+	@echo Built $(IMG):$(TAG) and tagged with latest
+
+.PHONY: push
```

Comment:
BTW: How can I publish this image ? Is it a manual process or done by some job in the CI?

Reply:
It's manual process. I think I will have to do that due permissions, just let me know when you are ready to push.

Reply:
Ok, I will. Thanks!